app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -489,6 +516,16 @@ def send_failed_fraud_email(user, enrollment)
     end
   end
 
+  def send_please_call_email(user, enrollment)
+    user.confirmed_email_addresses.each do |email_address|
+      # rubocop:disable IdentityIdp/MailLaterLinter
```

Comment:
This is a custom rule that enforces the use of `deliver_now_or_later` vs `deliver_later`. I'd wonder if we should be using `deliver_now_and_later` to avoid the lint? There are a few cases where we want to ensure it's delivered exclusively now or later, but I'm not sure that applies here.

https://github.com/18F/identity-idp/blob/main/lib/linters/mail_later_linter.rb

Reply:
Ok interesting. Thanks for that info @aduth. Yeah unfortunately I don't know enough about the motivation for enforcing that we do these calls async. This might be a good thing to bring back to our team and see what folks think.

Reply:
The idea with `deliver_now_or_later` is that we'd typically want to deliver messages asynchronously (to avoid blocking the response), but this is controlled through configuration to allow us the option to deliver synchronously if needed in an environment or if there's issues with asynchronous delivery.

See also:

https://github.com/18F/identity-idp/blob/21d5b412ea73b5006b7cb65738c3c706e40b41b4/config/initializers/ext_action_mailer.rb#L14-L20

Reply:
digging back through some old prs and it looks like we previously made the decision to use `deliver_later` for the emails sent in this job, [pr where we made the switch](https://github.com/18F/identity-idp/pull/7007)

Reply:
I generally don't like the pattern of `rubocop:disable` for stuff without a really compelling reason. But, as you pointed out, and as is found in some old Slack threads, there's precedent here: every other mailer call is `deliver_later`.

So, in the end, unless others object, I would support leaving this as-is. I'd rather the file stay consistent, even if it's not strictly necessary in this case.